CHANGELOG.md

```diff
@@ -13,6 +13,12 @@ As of 2025-08-29, changes are grouped as follows
 - 🐛 Bug Fixes: Bug fixes.
 - ⚡ Improvements: Transparent changes, e.g. better performance.
 
+## [Unreleased]
+
+### Features
+
+* **functions:** add `last_called` field to Function data class
+
```

Comment:
Changelog is updated automatically based on PR title. should not be updated manually.